netty/src/test/java/io/grpc/netty/AdvancedTlsTest.java

```diff
@@ -339,8 +339,8 @@ public void onFileReloadingKeyManagerTrustManagerTest() throws Exception {
     AdvancedTlsX509TrustManager serverTrustManager = AdvancedTlsX509TrustManager.newBuilder()
         .setVerification(Verification.CERTIFICATE_ONLY_VERIFICATION)
         .build();
-    Closeable serverTrustShutdown = serverTrustManager.updateTrustCredentialsFromFile(caCertFile,
-        100, TimeUnit.MILLISECONDS, executor);
+    Closeable serverTrustShutdown = serverTrustManager.updateTrustCredentials(caCertFile,100,
```

Comment:
Add a space before 100. Ditto elsewhere.

FWIW, I'll commonly linewrap before the 100, to keep it together with its unit. Although it seems you are purposefully doing the opposite, as late in the file you could have TimeUnit on the same line and you chose not to.

Reply:
Oh, missed the spaces because of IDE param name hints:
![Image 7-16-24 at 11 17 AM](https://github.com/user-attachments/assets/801a742f-4c8f-43ae-a01f-04668fc13690)
Will keep it disabled. 
I like the idea of having value + unit on the same line, also fixed it.